tests/src/ClientTest.php

```diff
@@ -17,6 +17,10 @@ class ClientTest extends TestCase
 {
     use ClientTestTrait;
 
+    /**
+     * @covers ::__construct
+     * @covers ::createContainer
```

Comment:
the container calls getConfigValue to fetch the main and auth endpoint config, since it doesn't explode I think we can say it covers this and extractConfigValues